hack/rpm-deps.sh

```diff
@@ -494,6 +498,24 @@ if [ -z "${SINGLE_ARCH}" ] || [ "${SINGLE_ARCH}" == "s390x" ]; then
         $centos_extra \
         $exportserverbase_main
 
+    bazel run \
```

Comment:
The lack of `--config=${ARCHITECTURE}` in both s390x and x86_64 pops to the eye but I don't know if really needed.